solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudSolrClientRetryTest.java

```diff
@@ -55,14 +56,17 @@ public void testRetry() throws Exception {
     solrClient.add(collectionName, new SolrInputDocument("id", "1"));
 
     ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set(CommonParams.QT, "/admin/metrics");
     String updateRequestCountKey =
         "solr.core.testRetry.shard1.replica_n1:UPDATE./update.requestTimes:count";
     params.set("key", updateRequestCountKey);
     params.set("indent", "true");
+    params.set(CommonParams.WT, "xml");
 
-    QueryResponse response = solrClient.query(collectionName, params, SolrRequest.METHOD.GET);
-    NamedList<Object> namedList = response.getResponse();
+    var metricsRequest =
+        new GenericSolrRequest(METHOD.GET, "/admin/metrics", SolrRequestType.ADMIN, params);
+    metricsRequest.setRequiresCollection(false);
```

Comment:
that's the default; can remove this line

Reply:
Done